test/e2e/functional/tests/e2e.go

```diff
@@ -32,33 +34,42 @@ import (
 
 var (
 	OperatorNameSpace = consts.DefaultOperatorNameSpace
-	TestIsOpenShift   = false
-	TestInterface     = "eth0"
-	TestIsSingleStack = false
+	retryInterval     = time.Second * 1
+	timeout           = time.Second * 120
```

Comment:
2 mins ? do we really need that long timeout ?

Reply:
No, but I set it at a large value to reduce flakes.